webapp/lib/view.dart

```diff
@@ -457,10 +457,29 @@ class ContentView {
           titleText: '$driverName',
           datasetLabels: List.filled(0, '', growable: true)
         );
+        driverChart.yUpperLimitRangeSlider.children[1].onChange.listen((e) {
+          var slider = (e.currentTarget as RangeInputElement);
+          var sliderIndicator = (e.currentTarget as Element).previousElementSibling;
+          if (int.parse(slider.value) == 0) {
+            controller.driverYUpperLimitFilters[driverName] = int.parse(slider.max);
+          } else {
+            controller.driverYUpperLimitFilters[driverName] = int.parse(slider.value);
+          }
+          controller.command(controller.UIAction.driverYUpperLimitSet, null);
+          var newValue = (int.parse(slider.value) - int.parse(slider.min)) * 100 / (int.parse(slider.max) - int.parse(slider.min));
+          var newPosition = 150 - (newValue * 2.15);
+          sliderIndicator.style.setProperty('top', 'calc(${-newValue}% + (${newPosition}px))');
+        });
         return driverChart;
       });
     });
   }
+
+  void setDriverChartsYAxisFilterMax(String driverName, num max) {
```

Comment:
I think this should also get called when we switch the period filter with the maximum for the sub-dataset visible? Otherwise it will get computed the first time, and then it could be either too small or too large for the other periods

Reply:
I see how that can be a bug. I've fixed it